transform/mattermost-analytics/models/intermediate/product/nps/int_nps_feedback.sql

```diff
@@ -0,0 +1,33 @@
+with mattermost_nps as (
+select 
+    distinct server_id as server_id
+    , user_id as user_id
+    , license_id as license_id
+    , event_date as event_date
+    , timestamp as timestamp
+    , server_version as server_version
+    , feedback as feedback
+    , user_role as user_role
+    , received_at as feedback_received_at
+    , CASE WHEN license_id = '{{ var('cloud_license_id') }}' THEN 'Cloud' 
+     WHEN license_id is not null and license_id <> '{{ var('cloud_license_id') }}' THEN 'Self-Hosted' END hosting_type
+    from {{ ref('stg_mattermost_nps__nps_feedback') }}
+), mm_plugin_prod as 
+(
+ select 
+    distinct server_id as server_id
+    , user_id as user_id
+    , license_id as license_id
+    , event_date as event_date
+    , timestamp as timestamp
+    , server_version as server_version
+    , feedback as feedback
+    , user_role as user_role
+    , received_at as feedback_received_at
+    , CASE WHEN license_id = '{{ var('cloud_license_id') }}' THEN 'Cloud' 
```

Comment:
Queries should be consistent in case. Let's use lowercase for all statements since this is what we use in the majority of the project.